modules/apigee_edge_teams/apigee_edge_teams.routing.yml

```diff
@@ -71,3 +71,51 @@ apigee_edge_teams.settings.team_app.cache:
     _title: 'Caching'
   requirements:
     _permission: 'administer team'
+
+apigee_edge_teams.redirect.team.add_form:
+  path: '/add-team'
+  defaults:
+    _controller: '\Drupal\apigee_edge\Controller\redirectController::manageRedirect'
+    redirect_route: 'entity.team.add_form'
+  requirements:
+    _permission: 'access content'
+
+apigee_edge_teams.redirect.team_app.collection:
+  path: '/team-apps'
+  defaults:
+    _controller: '\Drupal\apigee_edge\Controller\redirectController::manageRedirect'
+    redirect_route: 'entity.team_app.collection'
+  requirements:
+    _permission: 'access content'
+
+apigee_edge_teams.redirect.team_app.add_form:
+  path: '/team-apps/add'
+  defaults:
+    _controller: '\Drupal\apigee_edge\Controller\redirectController::manageRedirect'
+    redirect_route: 'entity.team_app.add_form'
+  requirements:
+    _permission: 'access content'
+
+apigee_edge_teams.redirect.team.add_members:
+  path: '/teams/{entity_slug}/add-members'
+  defaults:
+    _controller: '\Drupal\apigee_edge\Controller\redirectController::manageRedirect'
+    redirect_route: 'entity.team.add_members'
+  options:
+    parameters:
+      entity_slug:
+        type: entity:team
+  requirements:
+    _permission: 'access content'
+
+apigee_edge_teams.redirect.team_app.add_form_for_team:
+  path: '/teams/{entity_slug}/create-app'
+  defaults:
+    _controller: '\Drupal\apigee_edge\Controller\redirectController::manageRedirect'
```

Comment:
I think the issue the tests are failing is because the redirect controller should be uppercase on the class name.

Reply:
That was it. Thank you.